Makefile

```diff
@@ -1,5 +1,19 @@
 GOLANGCILINT ?= docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v1.57.2 golangci-lint
 DB_URL=postgres://postgres:12345@localhost:5432/postgres?sslmode=disable
+DB_URL_TEST=postgres://postgres:12345@localhost:5432/postgrestest?sslmode=disable
```

Comment:
NIT: While this will work, as a good practice I would recommend having an `.env_example` file with environment variables instead.